pkg/logs/config/config.go

```diff
@@ -123,15 +123,20 @@ func buildTCPEndpoints() (*Endpoints, error) {
 	}
 
 	var additionals []Endpoint
-	err := coreConfig.Datadog.UnmarshalKey("logs_config.additional_endpoints", &additionals)
+	var err error
+	raw := coreConfig.Datadog.GetString("logs_config.additional_endpoints")
+	if raw != "" {
+		err = json.Unmarshal([]byte(raw), &additionals)
+	} else {
+		err = coreConfig.Datadog.UnmarshalKey("logs_config.additional_endpoints", &additionals)
+	}
```

Comment:
Could we make this a helper function and avoid code duplication from line 165-171 ?

Reply:
Will do.